packages/@tinacms/cli/src/cmds/init/configure.ts

```diff
@@ -79,19 +79,236 @@ async function configure(
+    },
+    {
+      name: 'publicFolder',
+      type: (_, answers) =>
+        answers.framework.name !== 'next' && answers.framework.name !== 'hugo'
+          ? 'text'
+          : null,
+      initial: 'public',
+      message:
+        `Where are public assets stored? (default: "public")\n` +
+        logText(
+          `Not sure what value to use? Refer to our "Frameworks" doc: ${linkText(
+            'https://tina.io/docs/integration/frameworks/#configuring-tina-with-each-framework'
+          )}`
+        ),
+    },
+    {
+      name: 'forestryMigrate',
+      type: (_) => (env.forestryConfigExists ? 'confirm' : null),
+      initial: true,
+      message: `Would you like to migrate your Forestry templates?\n${forestryDisclaimer}`,
+    },
+    {
+      name: 'frontMatterFormat',
+      type: (_, answers) => {
+        if (answers.framework.name === 'hugo' && answers.forestryMigrate) {
+          if (env.frontMatterFormat && env.frontMatterFormat[1]) {
+            return null
+          }
+          return 'select'
+        }
+      },
+      choices: [
+        { title: 'yaml', value: 'yaml' },
+        { title: 'toml', value: 'toml' },
+        { title: 'json', value: 'json' },
+      ],
+      message: `What format are you using in your frontmatter?`,
+    },
+    {
+      name: 'overwriteTemplatesJS',
+      type: (_, answers) =>
+        !answers.typescript
+          ? env.generatedFiles['templates'].javascriptExists
+            ? 'confirm'
+            : null
+          : null,
+      message: `Found existing file at ${env.generatedFiles['templates'].javascriptExists}. Would you like to overwrite?`,
+    },
+    {
+      name: 'overwriteTemplatesTS',
+      type: (_, answers) =>
+        answers.typescript
+          ? env.generatedFiles['templates'].typescriptExists
+            ? 'confirm'
+            : null
+          : null,
+      message: `Found existing file at ${env.generatedFiles['templates'].fullPathTS}. Would you like to overwrite?`,
+    },
+  ]
+  // These questions are adding when running `tinacms init backend`
+  const backendSetupCommands: prompts.PromptObject[] = [
+    {
+      name: 'hosting',
+      type: 'select',
+      choices: [
+        {
+          title: 'Tina Cloud',
+          value: 'tina-cloud',
+        },
+        {
+          title: 'Self Host',
+          value: 'self-host',
+        },
+      ],
+      message:
+        'Do you want to use Tina Cloud to host your backend or self-host? (In self hosting you will have to bring your own auth, database and backend)',
+    },
+    {
+      name: 'githubToken',
+      type: (_, answers) => {
+        return answers.hosting === 'self-host' ? 'text' : null
+      },
+      message: `What is your GitHub Personal Access Token? (Hit enter to skip and set up later)\n${logText(
+        'Learn more here: '
+      )}${linkText(
+        'https://tina.io/docs/self-hosted/starters/nextjs-vercel/'
```

Comment:
@logan-anderson This should probably link to the /docs/self-hosted/existing-site/ link instead of the starter doc. (It looks like that doc may need to be updated to have some PAT context though)

Reply:
Updated the link.